android/src/androidTest/java/bbct/android/common/test/DatabaseUtil.java

```diff
@@ -22,12 +22,10 @@
 import android.content.Context;
 import android.database.Cursor;
 import android.database.sqlite.SQLiteDatabase;
-
-import java.util.List;
-
 import bbct.android.common.provider.BaseballCardContract;
 import bbct.android.common.provider.BaseballCardSQLHelper;
 import bbct.data.BaseballCard;
+import java.util.List;
```

Comment:
Is there a reason that this import was moved?

Reply:
I tried to modify stuff in this class and removed the list, and reintegrated it which caused this I think.

Reply:
This should be reverted back since the original code is the same as we get when using Alt-Ctrl-O (Organize Imports).